lib/winapi/synchapi.h

```diff
@@ -19,6 +20,15 @@ VOID EnterCriticalSection (LPCRITICAL_SECTION lpCriticalSection);
 BOOL TryEnterCriticalSection (LPCRITICAL_SECTION lpCriticalSection);
 VOID LeaveCriticalSection (LPCRITICAL_SECTION lpCriticalSection);
 
+VOID InitializeConditionVariable (PCONDITION_VARIABLE ConditionVariable);
+BOOL SleepConditionVariableCS (PCONDITION_VARIABLE ConditionVariable, PCRITICAL_SECTION CriticalSection, DWORD dwMilliseconds);
+BOOL SleepConditionVariableSRW (PCONDITION_VARIABLE ConditionVariable, PSRWLOCK SRWLock, DWORD dwMilliseconds, ULONG Flags);
+VOID WakeConditionVariable (PCONDITION_VARIABLE ConditionVariable);
+VOID WakeAllConditionVariable (PCONDITION_VARIABLE ConditionVariable);
+// DestroyConditionVariable is an Xbox extension to free associated system resources
```

Comment:
nit: This should make clear that it's an nxdk extension. "Xbox extension" could mean xboxkrnl.exe extension or XDK extension (or something along those lines).

Reply:
Changed it.